cves/kernel/CVE-2022-0516.yml

```diff
@@ -448,7 +464,12 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    This is likely a simple lapse. The function used for making the check on the virtual cpu was created
+    and intnded to be used, but the check was never placed in along with the rest. The fix was simply adding the check in.
+    CWE 280 recommends using "safe areas" surrounded by trust boundaries in the design of the subsystem. These were assumably 
+    already in place given the specificness of the other checks in the permissions functionality. The CWE also recommends always
+    checking that a resource was properlly and successfully accessed. This is now followed since such a check was added.
```

Comment:
Just a couple grammatical updates here as well that could help with readability, intended and properly